ios/Classes/SPSubject+Configuration.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  SPSubject+Configuration.swift
+//  snowplow_flutter_tracker
+//
+//  Created by DanielDPeter on 07.04.21.
+//
+
+import SnowplowTracker
```

Comment:
Is there a reason this file has a + in the file name? Seems like we would probably want to avoid special chars in filenames.

Reply:
It's just a common pattern in Swift development for files that contains extensions as they add (+) functionality to a Base class.